specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2020-07-01-privatepreview/examples/ServerKeyCreate.json

```diff
@@ -0,0 +1,43 @@
+{
+  "parameters": {
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "resourceGroupName": "testrg",
+    "serverName": "testserver",
+    "keyName": "someVault_someKey_01234567890123456789012345678901",
+    "api-version": "2020-07-01-privatepreview",
+    "parameters": {
+      "properties": {
+        "serverKeyType": "AzureKeyVault",
+        "uri": "https://someVault.vault.azure.net/keys/someKey/01234567890123456789012345678901"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/testrg/providers/Microsoft.DBForMySql/flexibleServers/testserver/keys/someVault_someKey_01234567890123456789012345678901",
+        "name": "omeVault_someKey_01234567890123456789012345678901",
+        "type": "Microsoft.DBforMySQL/flexibleServers/keys",
+        "kind": "azurekeyvault",
+        "properties": {
+          "serverKeyType": "AzureKeyVault",
+          "uri": "https://someVault.vault.azure.net/keys/someKey/01234567890123456789012345678901",
+          "creationDate": "2017-05-01T00:00:00.0Z"
+        }
+      }
+    },
+    "202": {
+      "body": {
```

Comment:
>body [](start = 7, length = 4)

This isn't actually a correct response for a 202. Please take a look at [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations) for more information. (I may have missed this from the other Swagger.)